ethereum/nft/trades/insert_seaport.sql

```diff
@@ -0,0 +1,148 @@
+CREATE OR REPLACE FUNCTION nft.insert_seaport(start_ts timestamptz, end_ts timestamptz=now(), start_block numeric=0, end_block numeric=9e18) RETURNS INTEGER
+LANGUAGE plpgsql AS $function$
+DECLARE r INTEGER;
+BEGIN
+
+WITH seaport AS (
+    SELECT block_number
+    , block_time
+    , buyer
+    , 'Buy' AS category
+    , currency_contract
+    , erc_standard
+    , erc_standard AS erc_types_array
+    , NULL AS erc_values_array
+    , evt_index
+    , 'Trade' AS evt_type
+    , exchange_contract_address
+    , nft_contract_address
+    , nft_contract_address AS nft_contract_addresses_array
```

Comment:
This pattern does not work I'm afraid. Same comment on lines 13, 22, 30 and 32.

NFT trades v1 aggregates multiple transfers in one row. Hence, the usage of an array.